src/main/java/de/tum/in/www1/artemis/service/connectors/localci/LocalCIBuildJobExecutionService.java

```diff
@@ -228,7 +226,8 @@ private LocalCIBuildResult runScriptAndParseResults(ProgrammingExerciseParticipa
             return constructFailedBuildResult(branch, assignmentRepoCommitHash, testRepoCommitHash, buildCompletedDate);
         }
 
-        List<String> testResultsPaths = getTestResultPaths(participation.getProgrammingExercise());
+        List<String> testResultsPaths = new ArrayList<>();
+        testResultsPaths.add("/repositories/test-repository/");
```

Comment:
this list only ever has one item, can you remove the list altogether and simply use the string?

Reply:
Good point! Just changed it accordingly.